### ISSUE TITLE
[SR-448] NSFileManager.attributesOfItemAtPath has wrong return value

### ISSUE THREAD

**Issue by drewcrawford** [apple/swift-corelibs-foundation#4178]
|                  |                 |
   |------------------|-----------------|
   |Previous ID       | SR-448      |
   |Radar             | None         |
   |Original Reporter | @drewcrawford      |
   |Type              | Bug    |
   |Status            | Resolved        |
   |Resolution        | Done    |
        
   <details>
  <summary>Additional Detail from JIRA</summary>

   |                  |                 |
   |------------------|-----------------|
   |Votes             | 0         |
   |Component/s       | Foundation    |
   |Labels            | Bug        |
   |Assignee          | None      |
   |Priority          | Medium      |

   

   md5: 1a4e4874dbb3de30b68d6cd448ee6fe7

  </details>

**Issue Description:**

At the moment, NSFileManager's `attributesOfItemAtPath` function [has the wrong return value](https://github.com/apple/swift-corelibs-foundation/blame/master/Foundation/NSFileManager.swift#L324). This appears to be a temporary workaround to another issue.

I am currently hacking around this incompatibility to make my software build; I wanted to get an actual bug \# in the tracker so that I can track it.

Comments suggest this is likely due to \[SR-138\], so this bug may be blocked by that issue.
